README.md

```diff
@@ -25,7 +25,7 @@ cp env.sample .env
 createdb webmaker
 
 # create tables
-psql -d webmaker -f migrations/create_webmaker_db.sql
+psql -d webmaker -f scripts/create-tables.sql
```

Comment:
It would be good to add an explicit "prerequisites" section to the docs that calls out Node 0.12 and the specific version of PG that we expect (9.4) with links to download / install instructions.